Running-Mastodon/Production-guide.md

```diff
@@ -48,6 +48,9 @@ server {
 
 
   add_header Strict-Transport-Security "max-age=31536000; includeSubDomains";
+  add_header Content-Security-Policy "default-src 'self'; img-src 'self' data:; connect-src wss://example.com 'self'; script-src 'self' 'unsafe-inline'; style-src 'self' 'unsafe-inline'";
```

Comment:
please add a visible line that tells the user they need to edit this `wss://example.com`, both in the config as a comment and in the doc below